Sage/carbon
# build(generate_proptypes): fix missing require default declaration

### Proposed behaviour

When adding the `prop-types` import into the transpiled component files in the `lib` folder, check if the declaration for `_interopRequireDefault` has already been added by Babel. If it hasn't, add it in manually.

Note we'll look to convert this script into a custom Babel plugin soon.

### Current behaviour

The script is adding in the `prop-types` import like this: 

```js
var _propTypes = _interopRequireDefault(require("prop-types"));
```
but not checking whether `_interopRequireDefault` has actually be declared anywhere in the file or not.

We have been lucky so far that there are normally default imports in our components. `VerticalDivider` however has no sole default imports and therefore `_interopRequireDefault` has not been added in by Babel.

### Checklist




#### QA


### Additional context



### Testing instructions

In master, after installing node_modules etc, run `npm run precompile` to run the build. 

Then check the contents of `lib/components/vertical-divider/vertical-divider.component.js`. You will see that the line
```js
var _propTypes = _interopRequireDefault(require("prop-types"));
``` 
has been added in.

This is not declared anywhere in that file so users will see the error: 
![image](https://user-images.githubusercontent.com/14963680/161262348-586efa82-dab2-4e45-a944-ae8801be6027.png)

Run the same as above on this branch and you will see that 
```js
function _interopRequireDefault(obj) {
  return obj && obj.__esModule ? obj : { default: obj };
}
```

has been added in `lib/components/vertical-divider/vertical-divider.component.js` to declare the function.
